src/Chapter1.hs

```diff
@@ -619,9 +628,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = last(div n 10) + last n 
```

Comment:
In Haskell, you usually add space between each argument, even those in brackets 🙂 
```suggestion
sumLast2 n = last( div n 10) + last n 
```